style.css

```diff
@@ -224,10 +224,16 @@ hr {
 	grid-row: 1;
 }
 
+.logo{
+	margin-right: 5vw;
+	text-align: right;
+}
+	
 .content_2{
 	box-sizing: border-box;
 	background-color: #f7f7f7;
 	background: rgba(204, 204, 204, 0.7);
+	border-radius: 5px;
```

Comment:
This is the border radius that is used to make the corners on some of the elements rounded.